packages/playwright-test/src/mount.ts

```diff
@@ -72,6 +74,24 @@ export const fixtures: Fixtures<
     },
   };
 
+const jsonType: Record<string, Function> = {
+  string: (value: any) => typeof value === 'string',
+  number: (value: any) => typeof value === 'number',
+  boolean: (value: any) => typeof value === 'boolean',
+  null: (value: any) => value === null,
+  array: (value: any) => Array.isArray(value) && value.every(isJson),
+  object: (value: any) => typeof value === 'object' && value !== null && !Array.isArray(value)
```

Comment:
JSON will have a default serializing conventions for types like Date, but I guess it is Ok to be more restrictive here.

Reply:
Look at what `JSON.stringify({ data: new Date() })` produces, but don't bother.